client/app/lib/components/contentModal/styl/contentModal.styl

```diff
@@ -275,11 +275,20 @@ main.main-container
     borderBox()
 
     .markdown-content
-      margin-top            10px
-      max-width             450px
-
-      pre code
-        overflow            auto
+      /* margin-top            10px */
+      /* max-width             450px */
+      padding                   10px 30px
+      
+      pre
+        margin                  0px
+        display                 inline-block
```

Comment:
you can get rid of this line, bc `inline-block` seems unnecessary here. `pre` elements default to `block` and that is fine.